xbmc/video/VideoDatabase.cpp

```diff
@@ -2072,8 +2072,6 @@ std::string CVideoDatabase::GetValueString(const CVideoInfoTag &details, int min
   std::vector<std::string> conditions;
   for (int i = min + 1; i < max; ++i)
   {
-    if (offsets[i].type == NULL || offsets[i].offset == NULL)
```

Comment:
the new type is not handled here and the offset (which is 0 for the UNUSED_TYPE) is also not checked. This worked by accident with the == NULL check.

How does that work now?